config/config.go

```diff
@@ -86,6 +87,8 @@ type SourceBlockchain struct {
 	BlockchainID                      string                           `mapstructure:"blockchain-id" json:"blockchain-id"`
 	VM                                string                           `mapstructure:"vm" json:"vm"`
 	RPCEndpoint                       string                           `mapstructure:"rpc-endpoint" json:"rpc-endpoint"`
```

Comment:
Let's use `APIConfig` to specify the URL, headers, and query parameters for `SourceBlockchain.RPCEndpoint`, `SourceBlockchain.WSEndpoint`, and `DestinationBlockchain.RPCEndpoint`. We currently use the same query params and headers for both WS and RPC connections, but the user may want these to be different.